iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimitsAdder.java

```diff
@@ -31,7 +30,7 @@ abstract class AbstractLoadingLimitsAdder<L extends LoadingLimits, A extends Loa
 
     public class TemporaryLimitAdderImpl<B extends LoadingLimitsAdder<L, B>> implements TemporaryLimitAdder<B> {
 
-        private static final Logger LOGGER = LoggerFactory.getLogger(ValidationUtil.class);
+        private final Logger logger = LoggerFactory.getLogger(TemporaryLimitAdderImpl.class);
```

Comment:
Why not static anymore?

Reply:
Sonar's rule stated I should use `TemporaryLimitAdderImpl` instead of `ValidationUtil` for the logger because it is its current class, but when I did I got this error : 

`[ERROR] /home/elcheikhnal/projects/powsybl-core/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimitsAdder.java:[33,70] non-static class com.powsybl.iidm.network.impl.AbstractLoadingLimitsAdder.TemporaryLimitAdderImpl cannot be referenced from a static context
`

Reply:
It is better to define the logger as a static attribute instead of as an instance attribute because there is no need to create a new one each time we want to add a new `TemporaryLimit`.
I suggest to declare it as a static attribute of `AbstractLoadingLimitsAdder` (and not in the inner class).

```java
private static final Logger LOGGER = LoggerFactory.getLogger(AbstractLoadingLimitsAdder.class);
```

This should solve both problems.